### PR TITLE
add object spread option

### LINKED ISSUE #739: Coverage is broken when using object spread operator
When executing lab with coverage and requiring a file within the test making use of object spread it fails with:

```
Unexpected token ...
```

Seems that there has been a similar bug in the past #514.

### PR DESCRIPTION
**Pull request by wzrdtales**
Maybe it would make sense to let users pass in their own config.

Fixes #739

### CODE REVIEW

**Comment by wzrdtales** [comment]
sry forgot to deactive prettier... overwritten now with an unfixed version

File: lib/coverage.js [hunk 1]
```diff
@@ -251,6 +251,9 @@ internals.instrument = function (filename) {
         loc: true,
         comment: true,
         range: true,
+        ecmaFeatures: {
+          experimentalObjectRestSpread: true
```
Comment (geek) [inline lib/coverage.js]: need to indent this a tad more, that is why the CI is failing

Comment (wzrdtales) [reply lib/coverage.js]: yeh before prettier died reindint just everything ;) Would be awesome if you could add an .eslintrc to your project so prettier can do that stuff for me :)

**Comment by wzrdtales** [comment]
Now it fails b/c it actually lints /home/travis/build/hapijs/lab/README.md oO

File: lib/coverage.js (hunk 1 above)
Comment (cjihrig) [reply lib/coverage.js]: https://github.com/hapijs/lab/blob/master/lib/linter/.eslintrc.js

Comment (wzrdtales) [reply lib/coverage.js]: @cjihrig That should be in the project root actually.

Comment (cjihrig) [reply lib/coverage.js]: Nonetheless, that's where it is if you want to access it.

**Comment by geek** [comment]
@wzrdtales published as 14.2.1

**Comment by wzrdtales** [comment]
+1

**Comment by temsa** [comment]
🎉

**Pull request merged**